bin/rename_fastq_headers_pre.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
+
+# Written by Jannik Seidel and released under MIT license.
+
+from Bio import SeqIO, bgzf
+import gzip
+import sys
+import json
+import argparse
+import re
+
+def parse_args(args=None):
+
+    parser = argparse.ArgumentParser(description="Renaming of the headers before running the core processes of the detaxizer pipeline.")
```

Comment:
Could this kind of "renaming of headers" perhaps be done (easily) with an already existing tool? If so, then that could perhaps be used in a later release of detaxizer?

Reply:
I'll do some more research on it, but so far, I have not found a solution that does exactly what is done in this script. If anybody has a good hint I would be happy to implement it in the future.